test/unit/template_invocation_task_result_map_test.rb

```diff
@@ -0,0 +1,25 @@
+require 'test_plugin_helper'
+
+describe TemplateInvocation do
```

Comment:
Seems like accidental file that doesn't belong to this PR? Tests are failing on `TemplateInvocation::TaskResultMap` not being defined.